ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -34,45 +37,44 @@ public void verifyModulesAvailableOnHomePage() throws Exception {
         assertTrue(homePage.isConfigureMetadataAppPresent());
         assertTrue(homePage.isSystemAdministrationAppPresent());
     }
-
+    
```

Comment:
not resolved, i still see it above @sherrif10

Reply:
This line is needed to make it more uniform than the other commits